app/assets/javascripts/discourse/tests/integration/components/sidebar/section-test.gjs

```diff
@@ -26,16 +29,21 @@ module("Integration | Component | sidebar | section", function (hooks) {
   });
 
   test("displaySection is dynamic based on argument", async function (assert) {
-    const template = hbs`
-      <Sidebar::Section
+    class State {
+      @tracked displaySection = false;
+    }
+    const state = new State();
+
+    const template = <template>
+      <Section
         @sectionName="test"
         @headerLinkText="test header"
         @headerLinkTitle="some title"
         @headerActionsIcon="plus"
         @headerActions={{this.headerActions}}
-        @displaySection={{this.displaySection}}/>`;
-
-    this.displaySection = false;
+        @displaySection={{state.displaySection}}
+      />
+    </template>;
     this.headerActions = [];
```

Comment:
Am I right in saying that we can't use `this.` at all in test `<template>`? If so, seems like `headerActions` should also be moved into the `state`? (even though it doesn't actually change)